cumulusci/tasks/metadata/metadata_map.yml

```diff
@@ -216,6 +220,26 @@ settings:
 scontrols:
     - type: Scontrol
       class: MetadataFilenameParser
+sharingRules:
+    - type: SharingCriteriaRule
+      class: MetadataXmlElementParser
+      extension: sharingRules
+      options:
+            item_xpath: './sf:sharingCriteriaRules'
+      - type: SharingOwnerRule
```

Comment:
Looks like there's a parse error because this line and line 234 are indented too far.

Reply:
sorry